pubspec.yaml

```diff
@@ -20,7 +20,7 @@ homepage: https://github.com/livekit/client-sdk-flutter
 
 environment:
   sdk: ">=3.6.0 <4.0.0"
-  flutter: ">=1.17.0"
+  flutter: ">=3.27.1"
```

Comment:
This is the min flutter version corresponding to SDK >=3.6

Reply:
I think it's `3.27.0` 🙂